jobs/integration/validation.py

```diff
@@ -1477,7 +1484,7 @@ async def test_keystone(model, tools, any_keystone):
     control_plane = model.applications["kubernetes-control-plane"]
 
     # save off config
-    config = await model.applications["kubernetes-control-plane"].get_config()
+    config = await control_plane.get_config()
```

Comment:
lol.  somebody was getting paid by the letter when they wrote that original line.